package.json

```diff
@@ -49,7 +49,7 @@
   },
   "dependencies": {
     "bluebird": "^3.3.3",
-    "contentful-batch-libs": "^4.9.2",
+    "contentful-batch-libs": "beta",
```

Comment:
Same as the [comment](https://github.com/contentful/contentful-import/pull/24#discussion_r107190991) in the import tool